packages/desktop-client/src/components/util/GenericInput.tsx

```
@@ -0,0 +1,443 @@
+export const GenericInput = forwardRef<HTMLInputElement, GenericInputProps>(
+  ({ style, ...props }, ref) => {
+    const dispatch = useDispatch();
+    const { isNarrowWidth } = useResponsive();
+    const { t } = useTranslation();
+    const { grouped: categoryGroups } = useCategories();
+    const dateFormat = useDateFormat() || 'MM/dd/yyyy';
+
+    let content: JSX.Element | null = null;
+    switch (props.type) {
+      case 'id':
+        const showPlaceholder = props.multi ? props.value.length === 0 : true;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)